wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -142,6 +143,20 @@ public static Command select(Map<Object, Command> commands, Supplier<Object> sel
     return new SelectCommand(commands, selector);
   }
 
+  /**
+   * Runs the command supplied by the supplier.
+   *
+   * @param supplier the command supplier
+   * @param requirements the set of requirements for this command
+   * @return the command
+   * @see DeferredCommand
+   */
+  public static Command defer(Supplier<Command> supplier, Set<Subsystem> requirements) {
```

Comment:
Rename to `deferred`?

Reply:
I used verbiage similar to the other factories (print, select, run, etc). Should the class be renamed to DeferCommand?